guides/spin/app/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: single
+title:  "Applications"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Overview
+
+Once you have `spin` [installed and configured](/guides/spin/cli), you can use it to start
+managing your Spinnaker applications.
+
+`spin` can manage the whole lifecycle of your application:
+
+```bash
+# NOTE: You can inspect any command with the `--help` flag.
+$ spin application --help
+
+This command is accessed by using one of the subcommands below.
+
+Subcommands:
+    delete    Delete the specified application.
+    get       Get the specified application.
+    list      List all applications.
+    save      Save the provided application.
+```
+
+The following assumes Spinnaker is running and Gate is
+listening on `http://localhost:8084`. If gate is running elsewhere,
+you can set the Gate endpoint with the global `--gate-endpoint` flag.
+
+## Managing Your Application's Lifecycle
+
+### Save
+
+You can create a new application with `save`:
+
+```bash
+$ spin application save --application-name my-app --owner-email jacobkiefer@google.com --cloud-providers "gce, kubernetes"
+
+Application save succeeded
+```
+
+Applications can also be updated with the `save` command.
+
+### List
+
+Let's list our applications in Spinnaker with `list`:
```

Comment:
Am I just listing applications (for display purposes) , or am I _getting a list_ (that is, can I do anything with the returned list as data)? 

The distinction between those two options goes to what text to use in the heading:
Show a list of applications
or 
Get a list of applications